pypeit/core/telluric.py

```diff
@@ -477,8 +580,8 @@ def tellfit(flux, thismask, arg_dict, init_from_last=None):
                 - ``arg_dict['flux_ivar']``:  Inverse variance for the
                   flux array
                 - ``arg_dict['tell_dict']``: Dictionary containing the
-                  telluric grid and its parameters read in by
-                  read_telluric_grid
+                  telluric model and its parameters read in by
+                  read_telluric_pca or read_telluric_grid
                 - ``arg_dict['ind_lower']``: Lower index into the
                   telluric model wave_grid to trim down the telluric
                   model.
```

Comment:
Add `, optional` to `init_from_last` on line 597.

Also, do we still need the `**kwargs_opt`?

Reply:
Done, and nope.